### ISSUE TITLE
invalid package

### ISSUE THREAD

**Issue by ImanCol** [Anatta336/driven-decals#3]
![image](https://user-images.githubusercontent.com/26169049/88494572-46dcef00-cf7c-11ea-9393-616b484a2b8b.png)

**Comment by Anatta336** [comment]
Hiya, thanks for the report.

It looks like the problem is Git isn't installed on the machine you're using. You can find a Windows installer here: [https://git-scm.com/download/win](https://git-scm.com/download/win). Once installed and Unity has been restarted the package manager should be able to use it to download the package correctly.

If you prefer not to install Git you can manually download the package [https://github.com/Anatta336/driven-decals/archive/master.zip](https://github.com/Anatta336/driven-decals/archive/master.zip). Unzip that somewhere and use Unity's package manager to "add package from disk...", selecting the `package.json` file in the unzipped directory. You will have to re-download to get future updates.

Hopefully that helps!

**Comment by ImanCol** [comment]
Yes I have Git, and GitHub Desktop. I had to manually download it from GitHub Desktop and import it locally into Package Manager.

**Comment by dbpienkowska** [comment]
Had the same issue - needed to add git.exe location to Windows PATH enviroment variable.
